styles/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+    background-color: #FFE1D6;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+ul {
+    list-style: none;
+}
+
+#main-menu {
+    margin: 10px;
+    width: auto;
+}
+
+#main-menu li {
+    display: block;
+    width: auto;
+    float: left;
```

Comment:
Try not to use `float` if you can help it. It has some side effects that can make the rest of your design line up like you expect.

Use `inline-block` instead to allow block elements to sit beside each other.